SpywareFilter/sections/tracking_servers_firstparty.txt

```diff
@@ -22,6 +22,7 @@
 ||tags.news.com.au^
 !
 ||pixelappcollector.thesun.co.uk^
+||sga.snapdeal.biz^
```

Comment:
Unsure about this url, when opening one you will get e. g. dresses and this is a shop.

https://sga.snapdeal.biz/pub/getContent?pubId=1&rs=html&impressionId=hello&slotId=sd-cr-198X232-1-3&siteId=101&js=true&response_type=similar_ad&min_val=4&max_val=4&inv=site&category=women-apparel&templateId=listing6&pagetype=clp&sdId=156458930472140737&q=&pincode=&eId=&zone=NO_ZONE&phn=null

Reply:
Request from issue generated by `sga.snapdeal.biz`. What about this rule? `||imp.snapdeal.biz/pub/imp?`

Reply:
That request is not made when I visit the site.

Reply:
<details><summary>Screenshot:</summary>

![2](https://user-images.githubusercontent.com/33488229/62411933-7ecc8300-b603-11e9-9c4a-d5fb14e4b91c.png)

</details><br/>

Reply:
`||imp.snapdeal.biz^` is ok